### DISCUSSION TITLE
Unstable training with nnconv on QM9

### DISCUSSION THREAD

**Discussion by xOmer12** [pyg-team/pytorch_geometric#10513] [Q&A]
I have been trying to use the nnconv on qm9 implementation from the repo for graph embeddings on QM9. However, when I train the model on some tasks (for example target 6 which is zpve), the training is extremely unstable - the loss on train seems to converge ok, but MAE on validation and especially on test can sometime grows to an insane value (1e10 and more). It can also return to be ok for like the very next epoch. I even tried to train with the vanilla implementation here without any modifiying, but this ain't stop. I have no idea how to solve it and if it actually suppose to be this way and im tripping over this.
Thx for help in advance!
